src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -667,11 +673,39 @@ private void spawnCitizen(CitizenData data)
 
             entity.setPosition(spawnPoint.getX(), spawnPoint.getY(), spawnPoint.getZ());
             world.spawnEntityInWorld(entity);
+            
+            checkAchievements();
 
             markCitizensDirty();
         }
     }
-
+    
+    /**
+     * Checks if the achievements are valid
+     * 
+     */
+    private void checkAchievements()
+    {
+        // Get the colonies owner UUID
+        final UUID uuidOwner = this.getPermissions().getOwner();
+        // the colonies size
+        final int size = this.citizens.size();
+        
+        EntityPlayer owner = ServerUtils.getPlayerOnServerFromUUID(uuidOwner);
+        
+        if (size >= AchSizeSettlement.SIZE) {
+            owner.triggerAchievement(ModAchievements.achSizeSettlement);
+        }
+        
+        if (size >= AchSizeTown.SIZE) {
+            owner.triggerAchievement(ModAchievements.achSizeTown);
+        }
+        
+        if (size >= AchSizeCity.SIZE) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)